### PR TITLE
Lesson_10

### PR DESCRIPTION
**Pull request by AlenaShurakova**

### CODE REVIEW

File: src/lesson_10/task_29/Runner.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        for (int i =0; i<10;i++){
+            marksList.add((int)(Math.random()*10));
+        }
+        System.out.println("Список всех оценок:" + marksList);
+        Iterator<Integer> iterator = marksList.iterator();
+        while (iterator.hasNext()) {
+            Integer s = iterator.next();
+            if (s.equals(0) || s.equals(1) || s.equals(2) || s.equals(3)) {
+                marksList.remove(s);
+                iterator = marksList.iterator();
```
Comment (RomanKashenok) [inline src/lesson_10/task_29/Runner.java:24]: you can just iterate with for cycle to not to create iterator

File: src/lesson_10/task_32/Runner.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.*;
+
+public class Runner {
+    public static void main(String[] args) {
+        String text = "Lorem lorem Ipsum -is simple, very very very dummy text!";
+        ArrayList<String> words = new ArrayList<String>(Arrays.asList(text.toLowerCase().replaceAll("\\p{Punct}+", " ").split("\\s+")));
+        System.out.println("Список слов:" + words);
+        Map<String, Integer> dictionary = new HashMap<>();
+        Iterator<String> iterator = words.iterator();
+        while (iterator.hasNext()) {
```
Comment (RomanKashenok) [inline src/lesson_10/task_32/Runner.java:16]: you can iterate one time over "words" and on every word get Value from dictinary. If value is null (such key is not in the map yet) you can pass 1 there, if value exasts - just add 1 to it.

**Review by RomanKashenok — CHANGES_REQUESTED** [review]

File: src/lesson_10/task_29/Runner.java (hunk 1 above)
Comment (AlenaShurakova) [reply src/lesson_10/task_29/Runner.java]: Fixed in "Lesson_10_Fixed" commit

File: src/lesson_10/task_32/Runner.java (hunk 2 above)
Comment (AlenaShurakova) [reply src/lesson_10/task_32/Runner.java]: Fixed in "Lesson_10_Fixed" commit

**Pull request merged**